content/doc/book/managing/nodes.adoc

```diff
@@ -1,6 +1,5 @@
 ---
 layout: section
-wip: true
```

Comment:
Please keep it. The section is not nearly ready. It documents 2 edge use-cases while it has no guidelines for common agent management use-cases

Reply:
added `wip:true` in header.